main.cpp

```diff
@@ -505,9 +609,23 @@ struct PlaneTail
     float heightUpperRudder = 62.f;
     int powerConsumed = 1;
     
-    void runPowerUnit (std::string);
-    void consumePower (bool);
-    void turnPlane (bool lowerRudderOpen = true);
+    void runPowerUnit(std::string);
+    void consumePower(bool);
+    void turnPlane(bool lowerRudderOpen = true);
+    void cancelRightTurn( int degreeOfrightTurnInducedByDrag, int rudderDegree)
+    {
+        auto dort = degreeOfrightTurnInducedByDrag;
+        while (rudderDegree <= 5)
+        {
+            for(dort = 1 ; dort <= 5; ++dort )
+            {
+            rudderDegree -= dort;
+            std::cout << "Rudder coordinates the plan to turn"<< std::endl;
+            }
+        break;
+        }
```

Comment:
Fix this indentation.
remove the `break`, because it prevents the `while` loop from looping.
- [ ] revise